build-runtime.sh.in

```diff
@@ -40,9 +50,10 @@ objcopy --add-section .sha256_sig=1024_blank_bytes \
 
 # Now statically link against libsquashfuse_ll, libsquashfuse and liblzma
 # and embed .upd_info and .sha256_sig sections
-$CC -o runtime "$repo_root"/elf.c "$repo_root"/notify.c "$repo_root"/getsection.c runtime3.o \
-    "$repo_root"/squashfuse/.libs/libsquashfuse_ll.a "$repo_root"/squashfuse/.libs/libsquashfuse.a \
-    "$repo_root"/squashfuse/.libs/libfuseprivate.a -L"$repo_root"/xz-5.2.3/build/lib \
+# TODO: squashfuse_LIBRARIES is a semicolon separated list of values that has to be split during runtime of this script
+squashfuse_libs="@squashfuse_LIBRARIES@"
+$CC $small_FLAGS $small_LDFLAGS -o runtime "$repo_root"/elf.c "$repo_root"/notify.c "$repo_root"/getsection.c \
+    runtime3.o ${squashfuse_libs//;/ } -L"@xz_LIBRARIES_DIR@" \
```

Comment:
Same here - where is `squashfuse_LIBRARIES` and `xz_LIBRARIES_DIR`pointing to and where is it coming from? Maybe we could at least leave a comment where these things are supposed to be on the systems we build on - CentOS and Debian/Ubuntu... (applies to all areas like this) - would help people like me a tremendous lot in case things are not working as expected

Reply:
Also I do not understand how `${squashfuse_libs//;/ }`works - please add a comment

Reply:
There is a comment already:

```
+# TODO: squashfuse_LIBRARIES is a semicolon separated list of values that has to be split during runtime of this script

Reply:
As mentioned below, this had to be merged now in order to fix the mess introduced last week which causes all builds to produce broken appimaged AppImages.